CHANGELOG.md

```diff
@@ -21,6 +21,47 @@ New Features
   });
   ```
 
+- Exposed SDK logger using [loglevel](https://www.npmjs.com/package/loglevel) module to allow for updating logging configuration after calling `Video.connect`, `Video.createLocalAudioTrack`, `Video.createLocalTracks` and `Video.createLocalVideoTrack`. With this feature, logs can now be intercepted at runtime to allow realtime processing of the logs which include but not limited to inspecting the log data and sending it to your own server.
+
+  ```js
+  var { Logger, connect } = require('twilio-video');
+  var token = getAccessToken();
+
+  // Connect using a custom logger name
+  connect(token, {
+    loggerName: 'logger-alice',
```

Comment:
👍  to de-emphasize `loggerName`. But I don't think we should deprecate `logLevel` since we don't plan on removing it, especially the ability to specify different levels for individual components.

Reply:
@charliesantos - 

Re: `logLevel` parameter:
This parameter is only there so that developer could specify individual logLevels for subModules right? I think we should deprecate/remove this parameter - as this feature is only useful for sdk developers and not useful for app developers. As @manjeshbhargav suggested, we can continue to support the parameter while its marked as deprecated.

Reply:
Understood @makarandp0 . I will mark it as deprecated.